core/src/test/java/org/opentosca/toscana/core/transformation/TransformationFilesystemDaoTest.java

```diff
@@ -24,41 +29,44 @@
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static org.opentosca.toscana.core.testdata.TestPlugins.PLATFORM1;
 import static org.opentosca.toscana.core.testdata.TestPlugins.PLATFORM2;
-import static org.opentosca.toscana.core.testdata.TestPlugins.PLATFORM4;
+import static org.opentosca.toscana.core.testdata.TestPlugins.PLATFORM3;
 import static org.opentosca.toscana.core.testdata.TestPlugins.PLATFORM_NOT_SUPPORTED;
 
 public class TransformationFilesystemDaoTest extends BaseSpringTest {
 
     @Autowired
     private TestCsars testCsars;
```

Comment:
Is this still used? Did not find any reference to this in code.

When removing this we could probably make this a Unit test too